BraveShared/tr.lproj/BraveShared.strings

```diff
@@ -0,0 +1,1353 @@
+
+/* Button title to close a menu. */
+"Close" = "Kapat";
+
+/* No comment provided by engineer. */
+"CloseAllTabsTitle" = "Tüm %i Sekmelerini Kapat";
+
+/* Code words input help */
+"CodeWordInputHelp" = "Sağlanan eşitleme zinciri kod sözcüklerinizi aşağıdaki forma yazın.";
+
+/* Code words section title */
+"CodeWords" = "Kod Sözcükleri";
+
+/* Text to choose for downloading a file (example: saving an image to phone) */
+"CommonDownload" = "indir";
+
+/* Text to select sharing something (example: image, video, URL) */
+"CommonShare" = "Paylaş";
+
+/* No comment provided by engineer. */
+"Confirm" = "Onayla";
+
+/* Cookie settings option title */
+"CookieControl" = "Çerez Denetimi";
+
+/* Settings item for clearing cookies and site data */
+"Cookies" = "Çerezler ve Site Verileri";
+
+/* Copied action complete title */
+"Copied" = "Kopyalandı";
+
+/* Copied codewords title */
+"CopiedToClipboard" = "Panoya kopyalandı!";
+
+/* Copy app info to clipboard action sheet action. */
+"CopyAppInfoToClipboard" = "Uygulama bilgilerini panoya kopyalayın.";
+
+/* Context menu item for copying an image to the clipboard */
+"CopyImageActionTitle" = "Resmi Kopyala";
+
+/* Context menu item for copying a link URL to the clipboard */
+"CopyLinkActionTitle" = "Bağlantıyı Kopyala";
+
+/* Copy codewords title */
+"CopyToClipboard" = "Panoya kopyala";
+
+/* Copy rewards internals info for support */
+"CopyWalletSupportInfo" = "Destek Bilgilerini Kopyala";
+
+/* Create PDF share action. */
+"CreatePDF" = "PDF Oluştur";
+
+/* Currently usedd search engines section name. */
+"CurrentlyUsedSearchEngines" = "Şu anda kullanılan arama motorları";
+
+/* Title for quick-search engines settings section. */
+"CustomSearchEngines" = "Özel Arama Motorları";
+
+/* Data Saved Shield Stat */
+"DataSavedStat" = "Tah. Veri \nKaydedilmiş";
+
+/* DuckDuckGo callout enable action. */
+"DDGCalloutEnable" = "Evet";
+
+/* DuckDuckGo message. */
+"DDGCalloutMessage" = "Özel arama ile Brave, bu Özel Sekmedeyken aramalarınızı yanıtlamak için DuckDuckGo'yu kullanacaktır. DuckDuckGo, arama geçmişinizi izlemeyen ve gizlilik içinde arama yapmanızı sağlayan bir arama motorudur.";
+
+/* DuckDuckGo callout no action. */
+"DDGCalloutNo" = "Hayır";
+
+/* DuckDuckGo callout title. */
+"DDGCalloutTitle" = "DuckDuckGo ile gizlilik içinde arama yapmak istiyor musunuz?";
+
+/* Button text to close the default browser popup. */
+"defaultBrowserCallout.introCancelButtonText" = "Şimdi değil";
+
+/* Button text to open app settings */
+"defaultBrowserCallout.introOpenSettingsButtonText" = "Ayarları Aç";
+
+/* Primary text on default browser popup screen */
+"defaultBrowserCallout.introPrimaryText" = "Gizliliğinizi korumak için Brave ile tüm bağlantıları açın";
+
+/* Secondary text on default browser popup. */
+"defaultBrowserCallout.introSecondaryText" = "Brave Kalkanları, izleyicileri ve reklamları engeller, veri kaydeder ve ziyaret ettiğiniz her sitede size zaman kazandırır";
```

Comment:
This is again suggesting we are saving data inside the application so we have to phrase it in a different way.

`"defaultBrowserCallout.introSecondaryText" = "Brave Kalkanları, izleyicileri ve reklamları engeller, veri kurtarır ve ziyaret ettiğiniz her sitede size zaman kazandırır";`